src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure implements FindArea {
+    private int firsLeg;
```

Comment:
The variable name `firsLeg` is misspelled. It should be `firstLeg` for clarity and consistency.